Content.Server/Light/Components/PoweredLightComponent.cs

```diff
@@ -66,6 +66,9 @@ public sealed class PoweredLightComponent : Component
 
         public CancellationTokenSource? CancelToken;
 
+        //Delay for the lights interactions
+        public TimeSpan cooldownEnd;
+
```

Comment:
Standard convention for public component variables is PascalCase. A more descriptive name like "CooldownBurnHand" would be more understandable at a glance too. The comment should be adjusted to reflect that it only affects burning the hand now.